### ISSUE TITLE
Is this project maintained?

### ISSUE THREAD

**Issue by Arctomachine** [Unitech/pm2#5650]
Not a bug report, but want to inquire about status of this project. Is it still maintained or is it already abandoned?

Numerous issues pile up in repo, yet not a single one of them gets commented by project owners, and they all get automatically closed by bot. Most notably bug reports in all forms and shapes about updating env variables not functioning.
Many merge requests with green check mark, some of them from few years ago, are not resolved.

So is this project still being worked on or is it better to start looking for alternatives?

**Comment by BuffMcBigHuge** [comment]
Sad to see no progress on PM2. It's a package used by countless production systems. I'm currently running the development branch to use `--stop-exit-codes`.

**Comment by csvan** [comment]
What even are the alternatives to pm2? I've been searching and the ones I can find are either themselves effectively abandoned (e.g. https://strong-pm.io/), or not functionally equivalent (e.g. nodemon)

**Comment by Arctomachine** [comment]
If I understood how it is done, I would suggest to make our own alternative. I can join in anyway, but with my current knowledge and understanding I have no idea how to design apps such as that.

**Comment by VeXell** [comment]
yep, sad news that no one is supporting this amazing tool. A huge amount of pool requests still in the backlog for a long time.

To solve it i wrote small tool to patch PM2 in my docker containers. This is only one way how to fix someting and improve it.

Hope one day maintain team will start work on it again.

**Comment by aoor9** [comment]
@Unitech
